libs/signal-store/src/lib/spec/feature-store.spec.ts

```diff
@@ -289,7 +296,7 @@ describe('FeatureStore', () => {
             template: ``,
         })
         class WelcomeComponent {
-            private cs = createComponentStore(counterInitialState);
+            private cs = createFeatureStore('someFeature', counterInitialState);
```

Comment:
Perhaps just name this variable `featureStore` since `cs` no longers covers the load if I understand correctly?

Reply:
renamed to `fs`